dockerfiles/cli/scripts/cmd_init.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) 2016 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+
+
+cmd_init_reinit_pre_action() {
```

Comment:
I think we had a typo there (reinit vs preinit) and it wasn't an issue as typo was in the hook name as well but now that the pre/post hooks are automated, is it an issue ?